ntoskrnl/mm/ARM3/pagfault.c

```diff
@@ -1614,12 +1614,13 @@ MiDispatchFault(IN BOOLEAN StoreInstruction,
 
 NTSTATUS
 NTAPI
-MmArmAccessFault(IN BOOLEAN StoreInstruction,
+MmArmAccessFault(IN ULONG FaultCode,
                  IN PVOID Address,
                  IN KPROCESSOR_MODE Mode,
                  IN PVOID TrapInformation)
 {
     KIRQL OldIrql = KeGetCurrentIrql(), LockIrql;
+    BOOLEAN StoreInstruction = MI_IS_WRITE_ACCESS(FaultCode);
```

Comment:
This is also a semantic change, albeit one that makes the code behave as it should. But perhaps start with "present fault" here and make it a "write access" check in its own commit for easier regression testing?